sql/parser/builtins.go

```diff
@@ -761,3 +773,73 @@ func datumToRawString(datum Datum) (string, error) {
 
 	return "", fmt.Errorf("argument type unsupported: %s", datum.Type())
 }
+
+// Round rounds x to n digits.
+// Directly taken from CPython's `double_round` in
+// cpython/Object/floatobject.c.
```

Comment:
I think we should stick with the `strconv.FormatFloat` version. Premature optimization is the root of all evil, right? If this ever looks like a performance issue we can revisit.